cli/info/info.go

```diff
@@ -0,0 +1,52 @@
+package info
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/cloudflare/cfssl/api"
+	"github.com/cloudflare/cfssl/api/client"
+	"github.com/cloudflare/cfssl/cli"
+	"github.com/cloudflare/cfssl/errors"
+
+	goerr "errors"
+)
+
+var infoUsageTxt = `cfssl info -- get info about a remote signer`
+
+var infoFlags = []string{"remote", "label", "profile", "config"}
+
+func bundlerMain(args []string, c cli.Config) (err error) {
+	if len(args) > 0 {
+		return goerr.New("argument is provided but not defined; please refer to the usage by flag -h.")
+	}
+
+	serv := client.NewServer(c.Remote)
```

Comment:
It would probably be useful to check whether `c.Remote` is empty, and let the user know if it is.